php-path-traversal/Config.php

```
@@ -0,0 +1,485 @@
+        if(!defined('DEV_ERRORS')) define('DEV_ERRORS',(defined('CANDY_DEVMODE') && CANDY_DEVMODE));
+        return new static();
+      }
+      public static function mail($m){
+        Config::masterMail($m);
+        return new static();
+      }
+    };
+  }
+
+  public static function key($k='candy', $stage=5){
+    $define = !defined('ENCRYPT_KEY') ? define('ENCRYPT_KEY', md5($k)) : false;
```

PHPMD (2.15.0)

Avoid unused local variables such as '$define'. (undefined)

(UnusedLocalVariable)